idk/kafka/cmd_test.go

```diff
@@ -682,3 +702,122 @@ func tDoHTTPPost(t *testing.T, url, contentType, body string) string {
 
 	return string(bod)
 }
+
+/*
+In some cases, when we see max.poll.interval.ms exceeded on the kafka consumer,
+we also see the consumer hang when it's trying to close properly. See SUP-288.
+This test:
+ 1. configures the consumer such that it will hang if max.poll.interval.ms is exceeded.
```

Comment:
I would like to see a bit more of an explanation on this -- what about our configuration would cause it to hang? Why would it hang? I don't know enough about the consumer to recognize how we did this.

Reply:
So this is a somewhat unanswered question. The consumer hangs in the librdkafka library that this https://github.com/confluentinc/confluent-kafka-go depends on. Which of course we depend on. A decision was made to accept that the consumer could hang and that we would just close it unsuccessfully if it did. So to answer this, we'd need to explore librdkafka as far as I know.